packages/amplify_authenticator/lib/src/mixins/authenticator_date_field.dart

```diff
@@ -0,0 +1,77 @@
+import 'package:amplify_authenticator/src/theme/amplify_theme.dart';
+import 'package:amplify_authenticator/src/widgets/form_field.dart';
+import 'package:intl/intl.dart';
+
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+
+final DateFormat formatter = DateFormat('yyyy-MM-dd');
```

Comment:
Make private, or static on AuthenticatorDateField, or rename to usDateFormatter or something and put in file for intl/formatters.

Reply:
Done